src/index.ts

```diff
@@ -38,7 +39,10 @@ app.use(keycloak.middleware());
 app.use(createUserIfNotExistsMiddleware);
 
 app.use("/api", helloRoutes);
+app.use("/api/sendEmail", emailRoutes);
```

Comment:
Maybe it should be better if no route is associated and juste use the function send email in the function in the back because the back can handle the notification system alone only by checking dates in the database
(not sure about that)